src/components/Referrals/referralsHelper.js

```diff
@@ -92,14 +109,13 @@ export function getUSDValue(value, decimals = 2) {
 }
 
 export function getCodeError(value) {
-  const trimmedValue = value.trim();
-  if (!trimmedValue) return "";
+  if (!value) return "";
```

Comment:
why do you think it's better to not use trimmed value?

Reply:
Sorry, I was thinking of spaces are included in the bytes32. I just checked spaces doesn't affect the bytes32.

I was thinking to show the error when there is any space in the input.

Reply:
I will bring that back.